src/components/Loader/PageLoader.js

```diff
@@ -2,8 +2,14 @@ import React from 'react'
 import cx from 'classnames'
 import styles from './PageLoader.module.scss'
 
-export const LoaderImage = () => (
-  <svg width='44' height='44' viewBox='0 0 32 32' fill='none' className={styles.loader__img}>
+export const LoaderImage = ({ withAnimation = true, width = 44 }) => (
+  <svg
+    width={width}
+    height={width}
```

Comment:
`height={width}` is a misleading assignment. If the `width === height` better to call the property `size`